src/shared/cache/MutationDataCache.ts

```diff
@@ -8,8 +8,8 @@ type Query = {
     entrezGeneId:number
 };
 
-function dataToKey(d:Mutation[]) {
-    return `${d[0].entrezGeneId}`;
+function dataToKey(d:Mutation[], entrezGeneId:number) {
+    return `${entrezGeneId}`;
```

Comment:
hmmmmm

Reply:
this is just fitting into the cache interface, the first argument is always the data. We need this change because `d` might be empty.

Reply:
at some point i will try to phase out use of this cache completely, and substitute `MutationCache` which I added, which is based on `MobxPromiseCache`

Reply:
this change turns out irrelevant because I ended up not using this cache in this PR.. but might as well keep it since it is a fix